packages/Molecules/Form/package.json

```diff
@@ -15,7 +15,6 @@
     "pikaday": "^1.7.0",
     "vue-form-generator": "^2.2.2",
     "vue-multiselect": "^2.1.3",
-    "vue-on-click-outside": "^1.0.3",
```

Comment:
Is that not required any more? @alan-cole @dylankelly ?

Reply:
That library was removed as it was causing issues with keyboard navigation. Dylan and I had discussed the possibility of including the custom implementation in a mixin - though I'm not sure what became of that.

Reply:
@alan-cole I will implement the mixin in #292